samples/excel/16-custom-functions/streaming-function.yaml

```diff
@@ -0,0 +1,32 @@
+order: 2
+id: excel-custom-functions-streaming
+name: Streaming function
+description: A streaming function that continuously increments the cell value.
+host: EXCEL
+api_set:
+    CustomFunctionsRuntime: 1.1
+script:
+    content: |
+        /** @CustomFunction 
+        *  @description Increments the cell with a given amount at a specified interval in milliseconds.
```

Comment:
Nit: the proper indentation for JSDOC looks like this:
<https://stackoverflow.com/questions/30676286/what-is-the-proper-canonical-formatting-of-long-jsdoc-lines>

Reply:
fixed.